src/main/java/com/compilercharisma/chameleonbusinessstudio/dto/Schedule.java

```diff
@@ -0,0 +1,43 @@
+package com.compilercharisma.chameleonbusinessstudio.dto;
+
+import java.util.*;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import lombok.*;
+
+/**
+ * A schedule represents a collection of repeating appointments, which can be
+ * collectively enabled or disabled
+ */
```

Comment:
Are the use cases of `Schedule` going to be for example, when a barber wants to block out certain time frames in the day and configure their own individual appointments throughout said time frame?

Reply:
sort of, as the app will automatically create individual appointments based upon the schedule. It's one of my next tasks, so it'll make more sense once I complete that, so here's my example psuedcode:
```
public void generateAppointmentsForSomeWeek(someWeek)
  for each enabled schedule in repository.getAll()
    for each enabled repeatingAppt (ra) in the schedule
       for each day the ra repeats on
         make a copy of the ra's appointment
         set the copy to occur on the given day on someWeek
```